oak_functions/loader/tests/module/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use anyhow::{anyhow, Context};
+use assert_matches::assert_matches;
+use maplit::hashmap;
+use std::collections::HashMap;
+
+type TestFn = fn(&str) -> ();
+
+struct TestManager<'a> {
+    tests: HashMap<&'a str, TestFn>,
+}
+
+impl TestManager<'static> {
+    fn new() -> Self {
+        Self {
+            tests: hashmap! [
```

Comment:
(nit) we usually prefer the { initialization for maps instead of [ (which is for lists)